src/test/java/org/xstefank/whitelist/RetestCommandTest.java

```diff
@@ -0,0 +1,28 @@
+package org.xstefank.whitelist;
+
+import java.util.ArrayList;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import static org.powermock.api.support.membermodification.MemberMatcher.method;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest(WhitelistProcessing.class)
+public class RetestCommandTest {
+
+    @Before
+    public void before() {
+        PowerMockito.stub(method(WhitelistProcessing.class, "loadCIs"))
```

Comment:
is this necessary now that CIs are configurable?

Reply:
I've just tried what you asked for.. But I think it does not make sense at now.
https://github.com/xstefank/tyr/pull/34#discussion_r238159236